### ISSUE TITLE
Option to control drill down option on date fields on a widget

### ISSUE THREAD

**Issue by bradzedonk** [RevealBi/Reveal.Sdk#320]
<img width="298" alt="Screenshot 2024-11-20 at 14 51 55" src="https://github.com/user-attachments/assets/7a85a806-75be-49a0-a330-a030b4e57d37">

Not all date fields work well with all drill down options thus it would be beneficial to control these on the widget.

**Comment by jberes** [comment]
@brianlagunas this might be a bug.  Even if I only choose Month for example, it gives me all drilldown options.  It should only give me what I have in the list.

**Comment by brianlagunas** [comment]
After discussing it with the team, we are considering this a bug. We will fix this to where there is no drill down unless you specify one.

**Comment by brianlagunas** [comment]
@bradzedonk I just want to clarify what you are asking for here. 

Are you asking for the auto drill down not to happen and for the drill down to only occur when you define a drill down? This is what Jason is considering a bug.

If not, could you elaborate on what you are asking for?

**Comment by bradzedonk** [comment]
@brianlagunas, Yes, that is correct. Not for it to auto drill down and the option to define on the widget which date options can be used.
